src/components/Settings/AllSettingsFields.tsx

```diff
@@ -246,10 +246,8 @@ export const AllSettingsFields = forwardRef(
                   to inject the version from package.json */}
               App version {APP_VERSION}.{' '}
               <a
-                onClick={openExternalBrowserIfDesktop(
-                  `https://github.com/KittyCAD/modeling-app/releases/tag/v${APP_VERSION}`
-                )}
-                href={`https://github.com/KittyCAD/modeling-app/releases/tag/v${APP_VERSION}`}
+                onClick={openExternalBrowserIfDesktop(RELEASE_URL)}
+                href={RELEASE_URL}
```

Comment:
@nadr0 Made the change here that we talked about on Slack. There are a few other places in the app that could benefit from the change but I don't want to blow up the scope of this quick one before the next nightly without being able to go through more extensive testing. I was able to test this locally, both for nightly and for regular.

Reply:
Thanks we can merge it!